forge-gui/res/cardsfolder/upcoming/lethal_throwdown.txt

```diff
@@ -0,0 +1,7 @@
+Name:Lethal Throwdown
+ManaCost:B
+Types:Sorcery
+A:SP$ Destroy | Cost$ B Sac<1/Creature> | ValidTgts$ Creature,Planeswalker | TgtPrompt$ Select target creature or planeswalker | SubAbility$ DBDraw | SpellDescription$ Destroy target creature or planeswalker.
```

Comment:
This templating is really awkward on WotC's part... should we do `Sac<1/creature or sacrifice a modified creature>` to match the Oracle??

Reply:
you need to adjust the syntax correctly